hunting/llm/rules/llm_dos_resource_exhaustion_detection.toml

```diff
@@ -0,0 +1,29 @@
+[hunt]
+author = "Elastic"
+integration = "aws_bedrock.invocation"
+uuid = "dc181967-c32c-46c9-b84b-ec4c8811c6a0"
+hypothesis = "Detects potential DoS attacks through abnormal token usage in LLM interactions."
```

Comment:
This seems like this should be the name and the `notes` the `description`. I would eliminate hypothesis, since that seems to be abstracted one layer up:

_I have an hypothesis, here are my hunt queries to validate it._. IOW, the queries are more atomic by design

Reply:
removed hypothesis.